server/src/main/java/org/opensearch/search/aggregations/bucket/terms/GlobalOrdinalsStringTermsAggregator.java

```diff
@@ -167,16 +167,18 @@ public void setWeight(Weight weight) {
         this.weight = weight;
     }
 
+    SortedSetDocValues getGlobalOrds(LeafReaderContext ctx) throws IOException {
```

Comment:
Unless I am missing something obvious all the callers have access to `valuesSource` 
So why not call it directly ?

Reply:
+1, we don't really need to abstract it out in a method

Reply:
So basically we have defined our `tryCollectFromTermFrequencies()` in GlobalOrdinalsStringAggregator class which needs globalOrds.

Now this globals ords changes in LowCardinality case. So either we pass a function consumer in the method and pass it when invoking the method which I think is less readable than what we have in this change.

So that's why I created a method `getGlobalOrds()` and then override it in LowCardinality class.

Reply:
Makes sense.